src/jats/jats-exporter.ts

```diff
@@ -925,8 +971,8 @@ export class JATSExporter {
         const xref = this.document.createElement('xref')
         xref.setAttribute('ref-type', 'bibr')
         xref.setAttribute('rid', normalizeID(rids.join(' ')))
-
-        if (citation.attrs.contents) {
+        xref.textContent = this.citationTexts.get(node.attrs.id) ?? ''
+        if (node.attrs.contents) {
           // TODO: convert markup to JATS?
           // xref.innerHTML = node.attrs.contents
           const text = textFromHTML(node.attrs.contents)
```

Comment:
This whole if statement should go away.

Reply:
Yes I'm still not done with this PR.

Reply:
I removed it, I'm also throwing an error if no match is found in the text map.